frontend/src/features/idir/search/components/IDIRPermitSearchResults.tsx

```diff
@@ -164,27 +166,30 @@ export const IDIRPermitSearchResults = memo(
           </Box>
         );
       },
-      renderRowActions: useCallback(({ row }: { row: MRT_Row<PermitListItem> }) => {
-        const isInactive =
-          hasPermitExpired(row.original.expiryDate) ||
-          isPermitInactive(row.original.permitStatus);
+      renderRowActions: useCallback(
+        ({ row }: { row: MRT_Row<PermitListItem> }) => {
+          const isInactive =
+            hasPermitExpired(row.original.expiryDate) ||
+            isPermitInactive(row.original.permitStatus);
 
-        if (shouldShowRowActions(idirUserDetails?.userAuthGroup)) {
-          return (
-            <Box className="idir-search-results__row-actions">
-              <IDIRPermitSearchRowActions
-                isPermitInactive={isInactive}
-                permitNumber={row.original.permitNumber}
-                permitId={row.original.permitId}
-                userAuthGroup={idirUserDetails?.userAuthGroup}
-                companyId={row.original.companyId?.toString()}
-              />
-            </Box>
-          );
-        } else {
-          return <></>;
-        }
-      }, []),
+          if (shouldShowRowActions(idirUserDetails?.userRole)) {
+            return (
+              <Box className="idir-search-results__row-actions">
+                <IDIRPermitSearchRowActions
+                  isPermitInactive={isInactive}
+                  permitNumber={row.original.permitNumber}
+                  permitId={row.original.permitId}
+                  userAuthGroup={idirUserDetails?.userRole}
```

Comment:
Maybe rename the prop name for "userAuthGroup" to "userRole" for IDIRPermitSearchRowActions component?